self-assessment.md

```diff
@@ -1,31 +1,144 @@
-# K8gb security self-assessment
+# k8gb Self-assessment
 
-2023-10-31
+Security reviewers: Yury Tsarev
```

Comment:
It makes sense to add @jkremser here as he originally contributed the self assessment